internal/unionstore/art_memdb.go

```diff
@@ -0,0 +1,265 @@
+package unionstore
```

Comment:
Is this file still needed?

Reply:
There are some benchmark comparations between `plar/go-adaptive-radix-tree` and ours in `memdb_bench_test.go`, it should be removed later.